### DISCUSSION TITLE
Exclude keywords?

### DISCUSSION THREAD

**Discussion by P51MustangMike** [ramdany7/RightClick-Folder-Icon-Tools#14] [Q&A] [answered]
Is there a way to set the "Generate from Keyword" option to exclude certain words from a title? the photo albums I'm working with contain 2 versions of a thumbnail with very similar names, and the only way I can think to filter out the undesirable one is by excluding a word from the keyword search.

Alternatively, in the UI it mentions _ as being a wildcard. If I could use these as part of the keywords this would solve my problem, as the cover image I want starts with an underscore!

Alternatively again, if I could just tell the software to simply always use the first image in a folder, that would work aswell! Are any of these things that I can do with the software? Its otherwise excellent and even without these features its still going to save me A LOT of time, so thanks for making this!

**Comment by ramdany7** [comment, marked as answer]
As of right now, there's no option to exclude certain keywords.

You can use the unreleased version of v0.4, which doesn't treat underscores (_) as wildcards anymore. Also, if you use `*.jpg` as a keyword, it will simply use the first .jpg image in the folder.

To download the unreleased version, just click on "Code" in this repository and then click "Download ZIP", or use this link:
https://github.com/ramdany7/RightClickFolderIconTools/archive/refs/heads/main.zip

Alternatively, you can copy the entire code from [this page](https://github.com/ramdany7/RightClickFolderIconTools/blob/main/RCFI%20Tools.bat) and replace the current version you're using.

Before using the unreleased v0.4, make sure to deactivate/uninstall v0.3 first by clicking "Uninstall.bat". Then, install v0.4 by opening "RCFI Tools.bat" which contains the v0.4 code.

**Reply by P51MustangMike** [reply]
The new version works perfectly with underscores! Thank you very much! You've just saved me a TON of time!!
